### PR TITLE
Get balance cadastre from balance file

### PR DESCRIPTION
**Pull request by rufener**

### CODE REVIEW

File: back/infolica/views/balance.py [hunk 1]
```diff
@@ -154,12 +128,21 @@ def get_balance_files_view(request):
...
         if filename.startswith(balance_filename_prefix) and (filename.endswith(".doc") or filename.endswith(".docx")):
-            files.append({'filename': filename, 'filepath': os.path.join(path, filename)})
-    
+            files.append({"filename": filename, "filepath": os.path.join(path, filename)})
+
     return files
 
 
-@view_config(route_name='balance_from_file', request_method='POST', renderer='json')
+def _formatNumeroBalance(cadastre_id, number):
```
Comment (kalbermattenm) [inline back/infolica/views/balance.py:136]: Shouldn't that not be camel cased?

File: back/infolica/views/balance.py [hunk 2]
```diff
@@ -168,56 +151,77 @@ def balance_from_file_view(request):
...
+    for p in doc.sections[0].header.paragraphs:
+        text = p.text.lower()
+        if text.startswith("cadastre"):
+            test = re.match("(cadastre (de|du) )", text.lower())
+
+            cadastre_ = text[test.span()[1] :]
+
+            cadastre = request.dbsession.query(
+                Cadastre,
+                func.similarity(Cadastre.nom, cadastre_).label("similarity")
```
Comment (kalbermattenm) [inline back/infolica/views/balance.py:172]: Isn't there a better (or more deterministic) way to get the cadastre?

File: front/src/components/Affaires/BalanceFromFile/BalanceFromFile.vue [hunk 3]
```diff
@@ -230,7 +230,7 @@ export default {
             response_data.push(...response_i.data)
           })
 
-          let relation = this.initRelationArray(response_data);
```
Comment (kalbermattenm) [inline front/src/components/Affaires/BalanceFromFile/BalanceFromFile.vue:233]: I do not get why this was suppressed...

Comment (rufener) [reply front/src/components/Affaires/BalanceFromFile/BalanceFromFile.vue:233]: This function `initRelationArray` was used to format the server response. But the webservice has been changed in order to serve directly the response in a correct format. This function is now useless and can be removed.

File: back/infolica/views/balance.py (hunk 1 above)
Comment (rufener) [reply back/infolica/views/balance.py:136]: Done.
I'm not sure to understand when to use or not to use camel case. But for code uniformity, I totally agree 👍

File: back/infolica/views/balance.py (hunk 2 above)
Comment (rufener) [reply back/infolica/views/balance.py:172]: It could be great, but I haven't found a better solution till now. I check it once more 👍

Comment (rufener) [reply back/infolica/views/balance.py]: At the moment, I haven't found a better solution...

**Pull request merged**